### DISCUSSION TITLE
Test mhz 74 and spectrum mode

### DISCUSSION THREAD

**Discussion by pietss** [egzumer/uv-k5-firmware-custom#303] [General]
Necessary premise: this is a limited number of tests, and for this reason I would like to know if there are any findings or not.
At approximately MHz 74, with the radio in the same position and the same antenna, reception from two repeaters:
1- weakened reception;
2- always in this frequency range, if I listen in VFO mode and switch to spectrum mode, this transmission is not detected (instead in the air band there seems to be no problem).

edit. this is what I observed with this latest version of the firmware, I might install the previous one again to check, maybe I will but as I said before I would also be interested in knowing about other people's tests

**Comment by ErikS-web** [comment]
Interesting. 

Do you receive Ham-repeaters or radio-stations ?

**Reply by pietss** [reply]
Fire brigade repeater. 
Could it be caused by a low battery charge level? I'm uploading them now and will try later

**Reply by iz3krk** [reply]
@pietss i have the same problem with fire brigade repeaeters with firmware after v.14

**Comment by pietss** [comment]
Spectrum mode, then VFO, then spectrum again (disappears). What you hear at the end of the video is at a different frequency. What you see does not always happen, other times after about 45 seconds the transmission is detected, furthermore it seems that with stronger signals this problem does not occur or is more rare 
Therefore, contrary to what I said yesterday, this also happens in the air band
https://mega.nz/file/ke1mmTjT#NWzynO_bP4sHcApTVTxZg6WzlzXSRIf_D9mhlP4QDiA

**Comment by pietss** [comment]
egzumer, will it be a solvable problem? 
Thank you

**Comment by egzumer** [comment]
1. the RSSI while receiving is higher that in scanning, this might be because scanning takes significantly less time to read RSSI and there is some accumulation effect when constantly listening on one freq.
2. trigger level is recalculated when spectrum starts.
3. Squelch in VFO is totally different thing than trigger level in spectrum.

Either way I see nothing wrong there. It's not like the signal disappeared. You can still see it on the graph.

**Comment by iz3krk** [comment]
For receive good 70 mhz band i need to use firmware v.13.
From firmware v.14 the reception of 70 mhz are not good. Need to increase RF gain

**Reply by egzumer** [reply]
It would be good if you made a video of how the issue looks.

**Reply by iz3krk** [reply]
ok, i have made 2 video on a fix carrier in 70 mhz band.  https://we.tl/t-ndp63CSVh7

@egzumer

**Comment by pietss** [comment]
LNA, IF, PGA, is it possible to make changes outside of spectrum mode (accessible in the Detail Monitor screen)?

**Comment by ErikS-web** [comment]
As mentioned in manual adjustable parameters are (only) possible in Detail Monitor screen of Spectrum analyzer.
